homeassistant/components/irm_kmi/utils.py

```diff
@@ -0,0 +1,33 @@
+"""Helper functions for use with IRM KMI integration."""
+
+import logging
+from typing import Any
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import CONF_LANGUAGE_OVERRIDE, LANGS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def get_config_value(config_entry: ConfigEntry, key: str, default: Any = None) -> Any:
+    """Get the value of key in the configuration.  If options were modified, they take priority."""
+
+    if config_entry.options and key in config_entry.options:
+        return config_entry.options[key]
+    if config_entry.data and key in config_entry.data:
+        return config_entry.data[key]
+    return default
```

Comment:
I am not sure what the benefit of this is. It's either in options or in data, not in both. So I think we should be specific instead of "Get it from somewhere"